source/conf.py

```diff
@@ -323,3 +324,19 @@ def new_send(self, data):
 
 
 http.client.HTTPConnection.send = new_send
+
+intersphinx_mapping = {
+    "robotpy": ("https://robotpy.readthedocs.io/en/stable/", None),
```

Comment:
Should be able to remove both of these and only link to https://robotpy.readthedocs.io/projects/robotpy/en/latest/ , which is the combined mostrobotpy api site. It doesn't include commands, but that can be added later if needed.